packages/ssr-web/deployment/fastboot-server.js

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable no-undef */
+const FastBootAppServer = require('fastboot-app-server');
+const fetch = require('node-fetch');
+
+let server = new FastBootAppServer({
+  distPath: 'dist',
+  gzip: true, // Optional - Enables gzip compression.
+  host: '0.0.0.0', // Optional - Sets the host the server listens on.
+  port: 4000, // Optional - Sets the port the server listens on (defaults to the PORT env var or 3000).
+  buildSandboxGlobals(defaultGlobals) {
+    // Optional - Make values available to the Ember app running in the FastBoot server, e.g. "MY_GLOBAL" will be available as "GLOBAL_VALUE"
+    return Object.assign({}, defaultGlobals, {
+      btoa: function (str) {
+        return Buffer.from(str).toString('base64');
+      },
+      fetch,
+    });
+  },
```

Comment:
In the future maybe we can find a way to share the sandbox global definition between this and the dev build?